css-fonts/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Fonts</title>
+  <link rel="stylesheet" href="styles.css"
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Fredoka&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Fredoka&family=Ubuntu&display=swap" rel="stylesheet">
```

Comment:
You only need the second set of `<link>`s here (which contains both fonts), not the first set (which contains only the Fredoka font).

Reply:
<img width="1678" alt="Screen Shot 2022-03-16 at 12 29 24 PM" src="https://user-images.githubusercontent.com/20695764/158674636-33c05b20-9ec2-4609-81fb-ba2a1eab89b2.png">

Reply:
Did I get rid of the correct set of `link`s?